### ISSUE TITLE
Some bullets have a wrong offset

### ISSUE THREAD

**Issue by bladeSk** [theflippedbit/opentyrian-nx#10]
After rebuilding the code with the latest devkitPro (and fixing it to be buildable - see https://github.com/theflippedbit/opentyrian-nx/pull/9), I discovered a major issue affecting the gameplay. Some of the shots are fired with an offset and/or under an incorrect angle. See the attached pictures.

I suspect this _might_ have something to do with the modified mouse code https://github.com/theflippedbit/opentyrian-nx/commit/eeed8639d8d6c7f2183841e0ced196a5227c6e1a#diff-ea38a79f5bdd7f3b040976ca4dc42261e52232cbb0eeac0d47ee3e64844d321b

Correct behavior:
![correct](https://user-images.githubusercontent.com/4263742/104489037-7c7ba000-55cf-11eb-8a9b-ca7894f9fdc9.png)

This build:
![incorrect](https://user-images.githubusercontent.com/4263742/104489069-856c7180-55cf-11eb-8115-dbec7cff54c0.jpg)

**Comment by bladeSk** [comment]
Yep, tried to build this commit https://github.com/theflippedbit/opentyrian-nx/commit/0cc978b69d8ceb1dde19887b78724233c76f5ae9 and the issue is gone.

**Comment by bladeSk** [comment]
Here's a fixed build, in case anyone was interested: https://github.com/bladeSk/opentyrian-nx/releases/tag/1.0.2
It also fixes the double keypress bug in menus.

**Comment by theflippedbit** [comment]
@bladeSk Nice!! I was sure I was bound to have messed something up removing that mouse logic.. I'd like to merge your fixes if you want to create a PR.
